### PR TITLE
Adds squidpeople

### PR DESCRIPTION
**Pull request by Wilchenx**
https://www.youtube.com/watch?v=Fmct2OAzE0g PR theme

## About The Pull Request

Adds a new playable squid species

image

Dedicated to squidqueen who complained about lack of them

Thanks to @WJohn for the amazing sprite!

Naming scheme:

Change the first letter of a human name
into Squ
unless the second letter is a o
then use Sqo

## Why It's Good For The Game

Something like this was never done before

Their weakness is not being able to wear shoes

## Changelog
:cl:
add: Squidpeople are a thing now
/:cl:

Todo:

Add low pressure weakness
Add high pressure resistance
Add ink spray ability
Add R'lyehian (https://www.yog-sothoth.com/wiki/index.php/R'lyehian)

### CODE REVIEW

File: strings/names/squid_male.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Sqoshua
+Squevin
+Squrian
+Squedward
+Squimothy
+Squason
+Squeffrey
+Squyan
+Squervere
+Nigward
```
Comment (tsaoToast) [inline strings/names/squid_male.txt]: speedmerge just for this alone

**Review by tsaoToast — APPROVED** [review]

**Comment by nervere** [comment]
:flushed:

File: strings/names/squid_male.txt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Squeorge
+Sqoshua
+Squevin
+Squrian
+Squedward
+Squimothy
+Squason
+Squeffrey
+Squyan
+Squervere
```
Comment (nervere) [inline strings/names/squid_male.txt]: very cool

**Review by nervere — APPROVED** [review]

**Comment by CthulhuOnIce** [comment]
Put squidpeople in the beestation folder and add it to `beestation.dme` instead of `tgstation.dme` and I will merge this mf

**Comment by CthulhuOnIce** [comment]
God help us

**Comment by Wilchenx** [comment]
BAD

**Comment by Wilchenx** [comment]
NO

**Comment by Wilchenx** [comment]
image

**Comment by Wilchenx** [comment]
I DIDN'T CLEANUP THE LAST FILE

**Comment by CthulhuOnIce** [comment]
Can't I merge again once you commit?

**Comment by Wilchenx** [comment]
i don't know

**Comment by Wilchenx** [comment]
commit is up

**Comment by CthulhuOnIce** [comment]
Just commit, worst case scenario I can open another PR from the same branch

**Pull request merged**